### PR TITLE
stable/concourse: Improve documentation awsSecretsManager template

### PR DESCRIPTION
**Pull request by shyamz-22**
Provide `correct` default template for pipeline and team for awsSecretsManager

Signed-off-by: Syamala Umamaheswaran <shykart2203@gmail.com>

### What
This PR updates `correct` default template for pipeline and team for awsSecretsManager. 
### Why
When we use the example template [`/concourse/{team}/{pipeline}/{secret}`], the concourse task fails with the following error:

`Invalid name. Must be a valid name containing alphanumeric characters, or any of the following: -/_+=.@!`

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by YoussB** [comment]
@shyamz-22, you need to bump the chart version here: 
https://github.com/helm/charts/blob/6ee98b392c12e31fe319265ab54dce484007f21d/stable/concourse/Chart.yaml#L3


Also, we are using the way we wrote these in a way to show that it uses the team name and not the word `{team}` or pipeline and so on, Do you think that this way is going to make things more clear for whoever reads the values file?

If so, can you also change the ones here: 
https://github.com/helm/charts/blob/6ee98b392c12e31fe319265ab54dce484007f21d/stable/concourse/values.yaml#L355-L361

**Comment by shyamz-22** [comment]
@YoussB Thank you for the reply. 

My motivation behind this change was wanted to have a different template value like `/concourse/a-unique-id/{team}/{pipeline}/{secret}`. We used exactly this value because of  the word choice `defaults to:` in the comment thinking that the default template value had this format.

Do you think it is a good idea to reword from `defaults to:`  to ` default template value:` to be more clear?

```    awsSecretsManager:

      ## pipeline-specific template for Secrets Manager parameters, default template value: /concourse/{{.Team}}/{{.Pipeline}}/{{.Secret}}
      ##
      pipelineSecretTemplate: 
```

**Comment by YoussB** [comment]
oh, I see. this makes sense!

**Comment by shyamz-22** [comment]
@YoussB I have couple more improvements to the documentation, do you think it makes sense to do it in the same PR or create a new one? Thanks :)

**Comment by YoussB** [comment]
@shyamz-22 Let's create a new one, so it won't get stuck on review, I will merge this one in 🥇

File: stable/concourse/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: concourse
-version: 7.0.0
```
Comment (YoussB) [inline stable/concourse/Chart.yaml]: @shyamz-22 you'll need to bump the version again.

**Review by YoussB — CHANGES_REQUESTED** [review]

**Comment by YoussB** [comment]
@shyamz-22 there is something weird `pull-charts-e2e` probably because your branch wasn't rebased, do you mind rebasing your branch on top of master.

**Comment by shyamz-22** [comment]
@YoussB Thanks for the hint. Should be fine now.

**Comment by shyamz-22** [comment]
synced with upstream again, cannot understand why tests keep failing

**Comment by YoussB** [comment]
for some reason, it is not able to deploy the chart, idk why 🤷‍♂

**Comment by YoussB** [comment]
although it did work before!!

**Comment by YoussB** [comment]
looks like the test environment used by prow is having problems:
```
I0628 20:46:18.534]   Warning  FailedScheduling   54s (x14 over 10m)      default-scheduler   pod has unbound immediate PersistentVolumeClaims (repeated 3 times)
```

**Pull request merged**